http.md

```diff
@@ -0,0 +1,340 @@
+# JSON RPC HTTP Methods
+
+The block engine accept HTTP requests using the [JSON-RPC 2.0](https://www.jsonrpc.org/specification) specification.
+
+## RPC HTTP Endpoint
+
+**Default port:** 443 e.g. [http://localhost:443/bundles](http://localhost:443/bundles), [http://192.168.1.88:443/bundles](http://192.168.1.88:443/bundles)
```

Comment:
nit: use `127.0.0.1` as IP example so people don't mistake these for actual IPs. also `https` for port 443?